docs/src/topics/flavors/cluster-autoscaler.md

```diff
@@ -0,0 +1,33 @@
+# Cluster Autoscaler
```

Comment:
Can we also point to the cluster autoscaler docs somewhere in here for additional reference? https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/FAQ.md